.github/workflows/ruby.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      DATABASE_URL: mysql2://root:@127.0.0.1:3306/wca_test
```

Comment:
When `DATABASE_URL` env variable is specified it overrides any default configuration, so if you specify a URL with the default GH Actions password like `mysql2://root:root@127.0.0.1:3306/wca_test`, it should work without altering the password.

Also, I think GH Actions have mysql 5.7 by default, but ideally we would run mysql 8 to mirror the production server. I think one option would be to use the `services:` configuration, which allows for running arbitrary docker images there.

Reply:
According to https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#mysql GitHub Actions runs mysql 8!

Reply:
Oh, perfect!!!